### ISSUE TITLE
[Insight] The Symfony Dependency Injection Container should not be passed as an argument - in src/…/Listener/GlsrRequestListener.php, line 63

### ISSUE THREAD

**Issue by Dev-Int** [GLSR/glsr#43]
**[in src/Glsr/GestockBundle/Listener/GlsrRequestListener.php, line 63](https://github.com/GLSR/glsr/blob/16a9297841d74ca54512161c1dfb22315493f88c/src/Glsr/GestockBundle/Listener/GlsrRequestListener.php#L63)**

> A Symfony dependency injection container has been found as an argument.

```php
     * @param \FrameworkBundle\Routing\Router $router    Routes of Request
     * @param array                           $routes    Routes to listen
     */
    public function __construct(
        EntityManager $etm,
        ContainerInterface $container,
        Router $router,
        $routes = array()
    ) {
        $this->etm = $etm;
        $this->container = $container;
```

---------------------

_Posted from [SensioLabsInsight](https://insight.sensiolabs.com/projects/de223fd6-2d6b-4433-a70a-08e09fa68510/analyses/62?status=violations)_

**Comment by Dev-Int** [comment]
Prepare an installation service to replace this Listener.

**Comment by Dev-Int** [comment]
Create a test when first access to Gestock's index #1 : 
 - [ ] if Company entity is empty -> add company 
 - [ ] if Settings entity is empty -> add settings 
 - [ ] if others settings (familylog, subfamilylog, zonestorage, unitstorage, tva) -> add others settings 
 - [ ] if Supplier entity is empty -> add supplier 
 - [ ] if Article entity is empty -> add article 
 - [ ] if Settings:firstinventory is empty -> create first Inventory #8 
For each item add button "Valid and add more" #29

**Comment by Dev-Int** [comment]
Delete GlsrRequestListener 32af1e76d27fd54aa2e70398e2ed7ad4de083c05

**Comment by Dev-Int** [comment]
Fix with 4a20e773d3dd5a127c592a1662efd11319a12eae
